num_list.py

```diff
@@ -19,5 +20,8 @@ def print_each_number_and_its_square():
     # The square of 10 is 100
     # ...etc
     # TODO - Write your code below this line.  (Keep the indentation)
-    
+    nums = [12, 10, 32, 3, 66, 17, 42, 99, 20]
+
+        for i in nums:
```

Comment:
Check indentations as it may be the reason your tests are failing.

Reply:
Thanks for the feedback, I hadn't realized it had failed! How far do I need to go back:
a) fork
b)edit
c) pull request

and if you know the steps I should take?

Reply:
No problem! And you don't need to fork again, you already have it. Just go back and edit your code again and then push it up again. The Pull Request will still be open and will automatically update. Then just tag me in the comments or something using `@`, like @Fiona-Gorrie, if you want me to review it again :) Please see me in class today if you want to go over this

Reply:
@TannerJuby1 got it! Thanks for the help!